### PR TITLE
Show total number of rows beside PageSize in the form of: [10] of 53

### PR DESCRIPTION
**Pull request by mbonaci**
Add classes to PageSize elements so they can be styled by the user

### CODE REVIEW

File: src/sematable.js [hunk 1]
```diff
@@ -181,6 +181,7 @@ const sematable = (tableName, TableComponent, columns, configs = {}) => {
           <div className="col-md-6">
             {showPageSize && <PageSize
               pageSize={pageInfo.pageSize}
+              totalSize={data.length}
```
Comment (mbonaci) [inline src/sematable.js]: Not 100% sure I'm passing the right thing here

File: src/PageSize.js [hunk 2]
```diff
@@ -2,27 +2,28 @@ import React, { Component, PropTypes } from 'react';
...
 class PageSize extends Component {
   render() {
     const {
       pageSize,
+      totalSize,
       onChange,
     } = this.props;
     return (
       <div
-        className="col-md-6"
```
Comment (mbonaci) [inline src/PageSize.js]: Not needed (`div` is already inside `col-md-6`)

**Comment by mbonaci** [comment]
Classes I added should be used like this:
image

**Comment by mbonaci** [comment]
The idea was to have:
image
when selection is not `All`
and:
image
when selection is `All`.

I don't know, maybe add `rows` or `total rows` to the end:
image

**Comment by amir-hadzic** [comment]
Thanks @mbonaci, I'll look into this.

**Comment by amir-hadzic** [comment]
Adjusted and merged in 3e1478d0dc22277dcc4f8346e71b8de93e454223, thanks Marko!

**Pull request closed without merging**